pages/index.js

```diff
@@ -212,16 +215,14 @@ export default function Home() {
   async function uploadPhoto(e) {
     const file = e.target.files[0];
     try {
-      await Storage.put(currentContact.id, file, {
+      const result = await Storage.put(currentContact.id, file, {
         contentType: "image/png",
       });
     } catch (err) {
-      console.log("Error uploading file: ", err);
+      console.log(err);
     }
-
-    const url = await Storage.get(currentContact.id);
-    setContactPhoto({ ...currentContact, photo: url });
-    // const result = await Storage.get(currentContact.id, { download: true });
+    const photoURL = await Storage.get(currentContact.id);
```

Comment:
You probably want to move these 2 lines inside the try block, since if it fails, then gets caught, this will run, and it will fail.